products/memgraph/upcoming/reference_guide/replication.md

```diff
@@ -0,0 +1,116 @@
+### How to Setup a Memgraph Cluster with Replication?
+
+Replication configuration is done primarily through openCypher commands. This
+allows the cluster to be dynamically rearranged (new leader election, addition
```

Comment:
Instead of `new leader election` use something similar to `changing the main instance`...